pages/nft/iscn/_iscnId.vue

```diff
@@ -266,11 +268,13 @@ export default class NFTTestMintPage extends Vue {
         case 'create': {
           const isAllowed = IS_TESTNET || await this.checkIsWhitelisted();
           if (!isAllowed) {
+            logTrackerEvent(this, 'IscnMintNFT', 'CheckIsWhitelistedError', ErrorType.USER_NOT_WHITELISTED, 1);
             this.toggleSnackbar(ErrorType.USER_NOT_WHITELISTED)
             break
           }
 
           if (!this.isUserISCNOwner) {
+            logTrackerEvent(this, 'IscnMintNFT', 'CheckIsWhitelistedError', ErrorType.USER_NOT_ISCN_OWNER, 1);
```

Comment:
I think a better log format would be set both error as `CreateNFTError` as `action` then `ErrorType.xx` as value
`GetAccountBalanceError` in above should be changed similarly